docs/examples/301/web-app-managed-identity-sql-db/.gitignore

```diff
@@ -0,0 +1,2 @@
+# ignore generated (transpiled) ARM templates, they should be treated like binaries
+*.json
```

Comment:
We actually require the JSON to be checked in (and validate that it matches the generated JSON) - it gives viewers of the repo the ability to see what the transpiled template looks like, plus also allows us to catch real-world regressions in generation.

Reply:
Some info on contributing examples here: https://github.com/Azure/bicep/blob/main/CONTRIBUTING.md#example-files